src/wallet/wallet.cpp

```diff
@@ -2501,10 +2502,11 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey, CCon
 
         if (fBroadcastTransactions)
         {
+            CValidationState state;
             // Broadcast
-            if (!wtxNew.AcceptToMemoryPool(maxTxFee)) {
+            if (!wtxNew.AcceptToMemoryPool(maxTxFee, state)) {
                 // This must not fail. The transaction has already been signed and recorded.
```

Comment:
Oh, is "must not" being used in the descriptive sense rather than prescriptive? If so, we should definitely re-write this.  

Also, I've encountered that specific error before so I'd agree with you.